CONTRIBUTING.md

```diff
@@ -28,10 +28,12 @@ Before you begin:
 - Once you submit your PR, others from the community will review it with you.
 - After that, we may have questions, check back on your PR to keep up with the conversation.
 
-### What goes where
-- The tutorials folder is for anythink showing you how to do, setup or use something.
-- The info folder is for any random information like good anticheat, scoreboard or ChatFilter plugins.
-- The recourses folder is for images for other files, configs etc.
+### Folder Conventions
+
+- The [tutorials folder](tutorials) is for any guides on how to setup/use something.
+- The [info folder](info) is for any information on commands, features etc of plugins, server jars etc.
+- The [resources folder](resources) is for images and other files.
+- Make sure you also put the contribution is in the correct language folder.
```

Comment:
Read Line 36 aloud

Reply:
You should be checking your commits before pushing them. Do they make sense?

Reply:
i swear i do. ill try to read like 10 times before i commit